code/game/objects/items/holy_weapons.dm

```diff
@@ -386,31 +386,35 @@
 	icon = 'icons/obj/transforming_energy.dmi'
 	icon_state = "swordblue"
```

Comment:
at some point, we really oughtta let people turn these things on and off.

but that's for another PR, I suppose.

Reply:
Transforming component.